Monika After Story/game/script-topics.rpy

```diff
@@ -12999,3 +12999,34 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_intrusivethought",category=['Psychology'],prompt="Intrusive thoughts",random=True))
+
+label monika_intrusivethought:
+    m 1euc "Hey [player]..."
+    m 1ekd "Have you ever had thoughts that you find upsetting, distressing, or all around unpleasant?"
+    m 3eub "I've been reading on a study on what's called 'intrusive thought', it's actually quite interesting." 
+    m 3ekc "The study involves that the mind tends to think of some...{w=0.5} Unpleasant thoughts when triggered by certain, often negative circumstances."
```

Comment:
"Unpleasant" doesn't need to be capitalized since it's part of the same sentence overall.